src/scripts/specialEvents/SpecialEvents.ts

```diff
@@ -39,99 +39,114 @@ class SpecialEvents implements Feature {
-            SeededRand.boolean() ? route.pokemon.land.push('Misdreavus') : null;
-        });
-        Routes.getRoutesByRegion(GameConstants.Region.hoenn).forEach(route => {
-            SeededRand.boolean() ? route.pokemon.land.push('Pikachu (Gengar)') : null;
-            SeededRand.boolean() ? route.pokemon.land.push('Shuppet') : null;
-            SeededRand.boolean() ? route.pokemon.land.push('Duskull') : null;
-        });
-    },
-    // End
-    new Date(new Date().getFullYear(), 10, 5, 23), () => {
-        Routes.getRoutesByRegion(GameConstants.Region.kanto).forEach(route => route.pokemon.land = route.pokemon.land.filter(p => !['Spooky Bulbasaur', 'Gastly'].includes(p)));
-        Routes.getRoutesByRegion(GameConstants.Region.johto).forEach(route => route.pokemon.land = route.pokemon.land.filter(p => !['Spooky Togepi', 'Misdreavus'].includes(p)));
-        Routes.getRoutesByRegion(GameConstants.Region.hoenn).forEach(route => route.pokemon.land = route.pokemon.land.filter(p => !['Pikachu (Gengar)', 'Shuppet', 'Duskull'].includes(p)));
-    }
-);
-// Let's Go P/E release date
-SpecialEvents.newEvent('Let\'s GO!', 'Encounter special Eevee and Pikachu roaming in the Kanto region.',
-    // Start
-    new Date(new Date().getFullYear(), 10, 16, 1), () => {
-        RoamingPokemonList.add(GameConstants.Region.kanto, new RoamingPokemon('Let\'s Go Pikachu'));
-        RoamingPokemonList.add(GameConstants.Region.kanto, new RoamingPokemon('Let\'s Go Eevee'));
-    },
-    // End
-    new Date(new Date().getFullYear(), 10, 23, 23), () => {
-        RoamingPokemonList.remove(GameConstants.Region.kanto, 'Let\'s Go Pikachu');
-        RoamingPokemonList.remove(GameConstants.Region.kanto, 'Let\'s Go Eevee');
-    }
-);
-// Christmas
-SpecialEvents.newEvent('Merry Christmas!', 'Encounter Santa Snorlax roaming the regions and discover the Grinch of Ilex Forest.',
-    // Start
-    new Date(new Date().getFullYear(), 11, 24, 1), () => {
-        // Add to every region excluding None
-        GameHelper.enumNumbers(GameConstants.Region).filter(i => i != GameConstants.Region.none).forEach(region => {
-            RoamingPokemonList.add(region, new RoamingPokemon('Santa Snorlax'));
-        });
-        dungeonList['Ilex Forest'].bossList.push(new DungeonBossPokemon('Grinch Celebi', 1600000, 100, {requirement: new GymBadgeRequirement(BadgeEnums.Elite_JohtoChampion)}));
-    },
-    // End
-    new Date(new Date().getFullYear(), 11, 30, 23), () => {
-        // Remove from every region excluding None
-        GameHelper.enumNumbers(GameConstants.Region).filter(i => i != GameConstants.Region.none).forEach(region => {
-            RoamingPokemonList.remove(region, 'Santa Snorlax');
-        });
-        dungeonList['Ilex Forest'].bossList = dungeonList['Ilex Forest'].bossList.filter(boss => boss.name != 'Grinch Celebi');
-    }
-);
-
+document.addEventListener('pokeclicker.game.initialized', () => {
```

Comment:
I think it would be better to move these to the `SpecialEvents.initialize()` function, so it's more consistent with how all the other `Feature` are handled.

Reply:
thanks for the hint, works flawlessly, but required a small refactoring tho (since SpecialEvent.ts called `initialize()` in their constructor)